eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -517,6 +527,14 @@ class Peer(val nodeParams: NodeParams,
         d.peerConnection forward unknownMsg
         stay()
 
+      case Event(store: PeerStorageStore, d: ConnectedData) if nodeParams.features.hasFeature(Features.ProvideStorage) && d.channels.nonEmpty =>
+        startSingleTimer("peer-storage-write", WritePeerStorage, nodeParams.peerStorageWriteDelay)
```

Comment:
This was on purpose. If a peer updates the storage all the time there is no need to persist it, better wait for the updates to stop.

Reply:
If we don't wait for the burst to be over, there is a high probability that we'll receive a new update just after a write, which means both more writes and more risk to lose funds for the user.
If the user sends updates continuously, even with your suggestion there is a high probability that if we stop eclair we won't have written the last update. And an outdated backup isn't better than no backup, it could even be worse.